aunt-leahs-app/src/components/tab/tab.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+
+// USAGE:
+{/* <TabComponent
+        name = "Tab"
+        items={[
+            { title: 'First Tab', content: <h1>hello</h1>},
+            { title: 'Second Tab',content: <CustomButton size="small" color="primary">I am a button</CustomButton>},
+            { title: 'Third Tab', content: 'Some other class'}
+            ...
+        ]} /> */}
+
+function TabPanel(props) {
+    const { children, value, index, ...other } = props;
+    return (
+        <Typography
+            component="div"
+            role="tabpanel"
+            hidden={value !== index}
+            id={`full-width-tabpanel-${index}`}
+            aria-labelledby={`full-width-tab-${index}`}
+            {...other}
+        >
+            <Box p={3}>{children}</Box>
```

Comment:
This is a magic number code smell. What is 3? This should probably be replaced by a variable.

Reply:
It is a padding for the tab content, i have created a variable name for it